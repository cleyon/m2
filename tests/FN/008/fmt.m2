@define FMT Format: AAA '%05d' BBB '%5s' CCC '%-5s'
@define A 42
@define B abc
@format FMT A B B@
