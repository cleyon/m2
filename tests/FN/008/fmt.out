Format: AAA '00042' BBB '  abc' CCC 'abc  '
